src/Text/Pandoc/Writers/OpenDocument.hs

```diff
@@ -684,14 +684,18 @@ listLevelStyle i =
                       , ("fo:margin-left", indent <> "in")
                       ]
 
-tableStyle :: Int -> [(Char,Double)] -> Doc Text
-tableStyle num wcs =
+tableStyle :: Int -> [(Text,Text)] -> [(Char,Double)] -> Doc Text
+tableStyle num kvs wcs =
     let tableId        = "Table" <> tshow (num + 1)
+        tableAttrStyles []   = []
```

Comment:
I think this could be simplified by using [`find`](https://hackage.haskell.org/package/base-4.14.0.0/docs/GHC-OldList.html#v:find) and [`maybeToList`](https://hackage.haskell.org/package/base-4.14.0.0/docs/Data-Maybe.html#v:maybeToList). Or [`mapMaybe`](https://hackage.haskell.org/package/base-4.14.0.0/docs/Data-Maybe.html#v:mapMaybe).

Of course, "simplified" here is subjective. I subscribe to the idea that explicit recursion is better avoided if possible, but YMMV.

Reply:
I changed it to a mapMaybe - thank you for the assistance.